rules-tests/CodeQuality/Rector/If_/ExplicitBoolCompareRector/Fixture/phpdoc-typed.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
could you rename the file with `skip_` prefix? thank you.

Reply:
Sure, missed that - thanks